src/main/kotlin/com/d3/chainadapter/adapter/ChainAdapter.kt

```diff
@@ -138,6 +138,20 @@ open class ChainAdapter(
         lastReadBlock.set(height)
     }
 
+    /**
+     * Returns raw query response with Iroha block.
+     * May be used to handle Iroha error codes manually
+     * @param queryAPI - Iroha queries network layer
+     * @param height - height of Iroha block to get
+     * @return Iroha block
+     */
+    private fun getBlockRawResponse(queryAPI: QueryAPI, height: Long): QryResponses.QueryResponse {
```

Comment:
why do you need it? queryAPI throws exceptions on errors

Reply:
Yep, you can use IrohaQueryHelperImpl.getBlock() and catch an exception instead these function.

Reply:
I think, there is no need in this function.

Reply:
Fixed